packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -293,6 +298,16 @@ export default Ember.Component.extend({
         'updateColumnOrder',
         newColumnOrder
       );
-    }
+    },
+
+    /**
+     * @action updateHeaderDisplayName
+     */
+    updateHeaderDisplayName(column) {
+      this.attrs.onUpdateReport(
+        'updateColumn',
+        column
```

Comment:
what happens if the display name is `""`?

Reply:
maybe mark the input field as invalid?

Reply:
Or we can have a placeholder to display default name?

Reply:
The consumer is coming in another PR right? We can do it in there.